app/views/batch_contexts/_new_bc_form.erb

```diff
@@ -43,11 +43,11 @@
                     as: :radio_buttons,
                     collection: [['Yes', true], ['No', false]] %>
                 <div data-ocr-target="runOcrImageNotes" hidden>
-                    <%= render 'alert', alert_type: 'warning', text: 'ocr_image' %>
+                    <%= render 'alert', alert_type: 'info', text: :ocr_image %>
                 </div>
                 <div data-ocr-target="runOcrDocumentNotes" hidden>
-                    <%= render 'alert', alert_type: 'info', text: 'ocr_document' %>
-                    <%= render 'alert', alert_type: 'warning', text: 'ocr_document' %>
+                    <%= render 'alert', alert_type: 'info', text: :ocr_document %>
+                    <%= render 'alert', alert_type: 'warning', text: :ocr_accessibility %>
```

Comment:
Line 49 should be info and ocr_accessibility
Line 50 should be warning and ocr_document